config/v1/types_oauth.go

```diff
@@ -33,15 +37,15 @@ type OAuthSpec struct {
 
 // OAuthStatus shows current known state of OAuth server in the cluster
 type OAuthStatus struct {
-	// TODO Fill in
+	// TODO Fill in with status of identityProviders and templates (and maybe tokenConfig)
+	// TODO we may need a list of Conditions to communicate failure
```

Comment:
unlikely

Reply:
Not committing to anything here.

Reply:
fine deferring this until we actually try to debug something.